change/react-native-windows-init-2020-06-24-03-38-49-winui3preview1.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "make cli strict and pass true to trailing bool param",
```

Comment:
What's our strategy for multiple change files? Do we want them, or should we just summarize changes in one, and rebase locally?

Reply:
@rectified95 these are changes in different packages so we need one change file for each